test/src/main/configurations/HTTP/ConfigurationWebServiceListenerSenderAttachments.xml

```diff
@@ -51,9 +51,9 @@
 				<forward name="success" path="GetFile"/>
 			</pipe>
 			<pipe
-				name="GetFile" className="nl.nn.adapterframework.pipes.FilePipe" actions="read" outputType="string" 
-				directory="${testdata.dir}/webservices" filename="filetext.txt" storeResultInSessionKey="attachment1" 
-				preserveInput="true" createDirectory="true"
+				name="GetFile" className="nl.nn.adapterframework.pipes.LocalFileSystemPipe" action="read" 
+				filename="${testdata.dir}/webservices/filetext.txt" storeResultInSessionKey="attachment1" 
+				preserveInput="true"
```

Comment:
please replace attribute `createDirectory` with `createFolder`, in stead of just deleting it.
(4 times)

Reply:
Attribute createFolder is for actions move, rename and list, here the action is read, is it still needed?